Dockerfile.exporter

```
@@ -1,12 +1,12 @@
 ARG BUILDPLATFORM=linux/amd64
-FROM --platform=$BUILDPLATFORM debian:bullseye-slim as importer
+FROM --platform=$BUILDPLATFORM golang:1.19.2-bullseye AS builder
 
```

`BUILDPLATFORM`, `TARGETOS`, and `TARGETARCH` and not build args unless `buildx` is used. For local testing purposes (`docker build` usage), we need to explicitly set these variables as ARGs. 

```suggestion
ARG TARGETOS=linux
ARG TARGETARCH=amd64
```